CTP/doc/sql/sql_guide.md

```diff
@@ -0,0 +1,524 @@
+* run test
+```
+======================================  Welcome to Interactive Mode ======================================  
+Usage: 
+    help         print the usage of interactive
+    run <arg>    the path of case file to run
+    run_cci <arg>    the path of case file to run_cci by cci driver
+    quit         quit interactive mode 
+
+For example:
+    run .                                                 #run the current directory cases
+    run ./_001_primary_key/_002_uniq                      #run the cases which are based on the relative path
+    run test.sql                                          #run the test.sql file
+    run /home/user1/dailyqa/trunk/scenario/sql/_02_object #run the cases which are based on the absolute path
+    run_cci ./_001_primary_key/_002_uniq                  #run the cases which are based on the relative path
+    run_cci ./_001_primary_key/_002_uniq/test.sql         #run the cases file
+sql>run .
+```
+
+![run_test](./ctp_run_test.png)  
+
+### csql  
+* cubrid createdb testdb en_US  
+* cubrid server start testdb
+* csql -u dba testdb  
+
+![run_test](./csql_run_test.png)  
+
+## 3.2 automatic monitoring configuration  
+**Test Machines**  
+
+No|role|user|ip|hostname|QUEUE_NAME|run_name
+:---:|:--:|:---:|:---:|:---:|:---:|:---:
+1|Test node|sql1|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_LINUX_GIT|run_sql
+2|Test node|sql2|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_LINUX_GIT|run_sql
+3|Test node|sql3|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_LINUX_GIT|run_sql
+4|Test node|sqlbycci|192.168.1.76|func01|QUEUE_CUBRID_QA_SQL_CCI_LINUX_GIT|run_sql_by_cci
+5|Test node|sql|192.168.1.77|func02|QUEUE_CUBRID_QA_SQL_PERF_LINUX|run_sql
+6|Test node|qa|192.168.1.161|winfunc01|QUEUE_CUBRID_QA_SQL_WIN64|run_sql
+
+* Enable configuration listening  
+  * touch start_test.sh  
+  ```
+  stop_consumer.sh 
+
+  prefix=`date "+%Y%m%d%H%M%S"`
+  cp nohup.out nohup.out.$prefix
+  echo "" > nohup.out
+
+  nohup start_consumer.sh -q [QUEUE_NAME] -exec [run_name] &
+  ```
+  * vi ~/.autoUpdate.sh  
+  ```
+  #!/bin/bash
+  export HOME=/home/[user]
+  export USER=[user]
+  if [ -f ~/.bash_profile ];
+  then
+            . ~/.bash_profile
+  fi
+  set -x
+  cd $HOME/CTP/common/sched/../script
+  chmod u+x *
+  ./stop_consumer.sh
+  ./upgrade.sh
+  nohup $HOME/CTP/common/script/start_consumer.sh -q [QUEUE_NAME] -exec [run_name]  2>&1 >> $HOME/nohup.out &
+  cd -
+  ```
+   * sh start_test.sh  
+* send test message  
+  login message@192.168.1.91 and send test message like:  
+  ```
+  sender.sh  [QUEUE_NAME]  [CI_BUILD] [Category] default
+  ```
+  [QUEUE_NAME]:Message queue name  
+  [CI_BUILD]:Corresponds to the build installation package address
+  [Category]:sql,sql_debug,sql_by_cci,medium,medium_debug  
+# 4. Verify test Results  
+How to view qahome results:
+
+![qa_result](./qa_result.png)  
+
+**Validation step**  
+* Step 1: bug recurrence  
+Use CTP and CSQL tools to reproduce the test
+* Step 2: analyze the cause of the bug  
```

Comment:
How to revise the test cases/answers if it is a revise required issue? 
How to make/add a sql test case? How to make a sql_by_cci answer? How about windows answer?
What should the user pay attention to when they write a test case?
What is the naming rule of sql test cases? (for a bug/for new feature)
How to open a regression/crash issue? Give some examples.